Changelog.md

```diff
@@ -2,7 +2,8 @@
 
 __New Features__
 - HVAC modeling updates:
-  - **Breaking Change**: `CompressorType` required for central and mini-split air conditioners and heat pumps.
+  - **Breaking Change**: `CompressorType` required for central and mini-split air conditioners and heat pumps as well as ground-to-air heat pumps.
+  - Optional input `SimulationControl/AdvancedResearchFeatures/GroundToAirHeatPumpModelType` to choose "simple" (default) or "advanced"; "advanced" ground-to-air heat pump model better accounts for coil staging.
```

Comment:
How about if we change "advanced" to "experimental"? That would be a small change to the code, and I think it would be a sufficient warning to a user that they probably shouldn't choose this yet unless they have a good reason for it.